### ISSUE TITLE
Trained keypoint detection model with YOLOv3 detections instead of GT boxes

### ISSUE THREAD

**Issue by shreyasrajesh** [stefanopini/simple-HRNet#9]
Hi,

I just wanted to check with you if you have or plan to retrain a new HRNet model for the pose keypoint detection using the YOLOv3 detection boxes as the ground truth instead of the COCO annotated ground truth boxes used to train the original model. I feel like the model might learn to handle keypoint detection in noisy/wrong YOLO boxes to a greater extent as compared to the current model. Do you believe there is any scope for this and it may help the accuracy of the pose points that are detected ?

**Comment by stefanopini** [comment]
Hi,
That's a nice idea!
I do believe it could improve the model performance, but I'm not currently planning to re-train the network using the yolo detection boxes.
However, I'm going to add the training code to this repository by the end of July (hopefully).

**Comment by shreyasrajesh** [comment]
Great!
The training code will be really helpful. Looking forward to it.

**Comment by stefanopini** [comment]
Hi @shreyasrajesh ,
Training code is now available. I hope it helps.

**Comment by shreyasrajesh** [comment]
Thank you very much for your timely update!
